.github/workflows/integ_test_api_graphql_auth_directive.yml

```diff
@@ -1,6 +1,22 @@
 name: Integration Tests | API - GraphQL Auth Directive
 on:
   workflow_dispatch:
+    inputs:
+      ios:
+        description: '📱 iOS'
+        required: true
+        default: true
+        type: boolean
+      tvos:
+        description: '📺 tvOS'
+        required: true
+        default: true
+        type: boolean
+      watchos:
+        description: '⌚️ watchOS'
```

Comment:
is there no integration test for watchOS?

Reply:
Thanks for catching these - addressed in [457931f](https://github.com/aws-amplify/amplify-swift/pull/3318/commits/457931f2c406914c6dc6409d3de17cc2c818225d)